jsonnet/main.jsonnet

```diff
@@ -294,7 +294,7 @@ local inCluster =
       thanosQuerier: $.values.thanos {
         name: 'thanos-querier',
         namespace: $.values.common.namespace,
-        replicas: 2,
+        replicas: 3,
```

Comment:
with `minAvailable: 1` we don't need to increase the number of replicas

Reply:
Thanks, I will update both parts accordingly 👍